mjpc/tasks/humanoid/tracking/task.cc

```diff
@@ -0,0 +1,229 @@
+// Copyright 2022 DeepMind Technologies Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "tasks/humanoid/tracking/task.h"
+
+#include <algorithm>
+#include <array>
+#include <cassert>
+#include <cmath>
+#include <cstring>
+#include <iostream>
+#include <map>
+
+#include <mujoco/mujoco.h>
+#include "../../../task.h"
+#include "utilities.h"
+
+
+static std::tuple<int, int, double, double> compute_interpolation_values(
```

Comment:
Nit: Make this more C++-y:

Put it in an anonymous namespace (instead of `static`), and name it ComputeInterpolationValues.

Reply:
Thanks! I'm a bit noob with C++ so wasn't sure what the best practice here was. Should be fixed now.